router/index.js

```diff
@@ -1,47 +1,103 @@
 'use strict';
 
 const express = require('express');
+const db = require('../server/db');
 
 const router = express.Router();
 
+// Home route
+router.get('/', async (req, res, next) => {
+  try {
+    // Get all the sun shade porducts json
+    const sunshades = await db.getSunshades();
+
+    // Send status and render home view
```

Comment:
Home route successfully renders the table populated with all products in the json. Code is clearly commented and easy to follow. Returns an appropriate status code, and has try and catch used. Great job!